server/controllers/stock/index.js

```diff
@@ -195,29 +204,25 @@ function normalMovement(document, params) {
  * @description movement between depots
  */
 function depotMovement(document, params) {
-  let paramIn;
-  let paramOut;
   let isWarehouse;
   const transaction = db.transaction();
   const parameters = params;
-  const isExit = parameters.isExit;
+  const isExit = parameters.isExit ? 1 : 0;
 
   let record;
-  
+
   parameters.entity_uuid = parameters.entity_uuid ? db.bid(parameters.entity_uuid) : null;
 
-  const depot_uuid = isExit ? db.bid(parameters.from_depot) : db.bid(parameters.to_depot);
-  const entity_uuid = isExit ? db.bid(parameters.to_depot) : db.bid(parameters.from_depot);
-  const is_exit = isExit ? 1 : 0;
-  const flux_id = isExit ? core.flux.TO_OTHER_DEPOT : core.flux.FROM_OTHER_DEPOT;
+  const depotUuid = isExit ? db.bid(parameters.from_depot) : db.bid(parameters.to_depot);
+  const entityUuid = isExit ? db.bid(parameters.to_depot) : db.bid(parameters.from_depot);
+  const fluxId = isExit ? core.flux.TO_OTHER_DEPOT : core.flux.FROM_OTHER_DEPOT;
 
   parameters.lots.forEach((lot) => {
-
     record = {
-      depot_uuid,
-      entity_uuid,
-      is_exit,
-      flux_id,
+      depot_uuid    : depotUuid,
```

Comment:
Using ```{depot_uuid}``` is valid like using ```{depot_uuid : depotUuid}``` so why to remove it?

Reply:
By rules define by eslint we must use `camelCase` practice for all variable name. And `depotUuid` is a variable name.